### PR TITLE
common stats, incr and timer

### PR DESCRIPTION
**Pull request by pnuckowski**

### CODE REVIEW

File: tests/test_stats.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    @patch('statsd.StatsClient.incr')
+    def test_increment_decorator(self, incr_mock):
+        self.stat.decorated_incr()
+
+        self.assertTrue(incr_mock.called)
+
+    @patch('statsd.StatsClient.incr')
+    def test_increment_context(self, incr_mock):
+        with stats.increment():
```
Comment (benethor) [inline tests/test_stats.py]: can't we just call stats.increment()?

File: tests/test_stats.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        timer_klass = stats.timer.start('test_manual_timer')
+        timer_klass.stop()
+
+        self.assertTrue(timer_mocked_klass.stop.called)
+
+    @patch('statsd.StatsClient.timer')
+    def test_timer_context_manager(self, timer_mock):
+        timer_mocked_klass = MagicMock()
+        timer_mock.return_value = timer_mocked_klass
+        with stats.timer(metric_name='test_timer_context'):
```
Comment (benethor) [inline tests/test_stats.py]: same as above

File: tests/test_stats.py (hunk 1 above)
Comment (pnuckowski) [inline tests/test_stats.py]: No we can't

File: tests/test_stats.py (hunk 2 above)
Comment (pnuckowski) [inline tests/test_stats.py]: same as above

**Pull request merged**